### DISCUSSION TITLE
What are the ports to be port forwarded in the firewall for using WebRTC stream

### DISCUSSION THREAD

**Discussion by BharadwajManne** [bluenviron/mediamtx#2004] [Questions] [answered]
What are the ports to be port forwarded in the firewall for using WebRTC stream.

**Comment by aler9** [comment, marked as answer]
Hello, the only port you have to forward is TCP 8889, that provides the web server that allows to initialize a WebRTC connection with the server. WebRTC uses a mechanism called ICE to establish a UDP connection between two peers, that doesn't require firewall changes.

This is valid unless you have connectivity issues and you need to use `webrtcICEUDPMuxAddress` or `webrtcICETCPPMuxAddress` as described in the README, in this case you have to forward ports of `webrtcICEUDPMuxAddress` and `webrtcICETCPPMuxAddress` too.

**Reply by BharadwajManne** [reply]
Thank you.

[a comment by an automated account is not shown]
